application/features/bootstrap/MfaContext.php

```diff
@@ -126,6 +155,50 @@ public function theUserDoesNotHaveAManagerEmailAddress()
         $this->theResponseStatusCodeShouldBe(200);
     }
 
+    /**
+     * @Given the user has requested a new webauthn MFA
+     */
+    public function theUserHasRequestedANewWebauthnMfa()
+    {
+        $user = User::findOne(['employee_id' => $this->tempEmployeeId]);
+        Assert::notEmpty($user, 'Unable to find that user.');
+        $this->setRequestBody('type', Mfa::TYPE_WEBAUTHN);
+        $this->iRequestTheResourceBe('/mfa', 'created');
+
+        $id = $this->getResponseProperty('id');
+        Assert::notEmpty($id, 'Unable to get id of new Webauthn MFA');
+        $mfa = Mfa::FindOne(['id'=>$id]);
+        Assert::notEmpty($mfa, 'Unable to find that MFA.');
+
+        // It is too complicated at this point to come up with completely correct values
+        // These should get as far as producing a 400 status code with
+        // "error":"unable to create credential: Error validating challenge"
+
+        // These values are from values produced in serverless-mfa-api-go/webauthn_test.go Test_FinishRegistration
+        $reqValue = [
+            'id' => 'dmlydEtleTExLTA',
+            'rawId' => 'dmlydEtleTExLTA',
+            'type' => 'public-key',
+            'response' => [
+                'authenticatorData' => 'hgW4ugjCDUL55FUVGHGJbQ4N6YBZYob7c20R7sAT4qRBAAAAAAAAAAAAAAAAAAAAAAAAAAAADHZpcnR1YWxrZXkxMaQBAgMmIVggBtYaQhitMvmuvKeeUZmuh96TmXTRGxB_6bfslWmTVF4iWCCK1h-O_T8R6MjkIWCsX-Pry8RJhuOxbDwovnYJBu0SZw',
+                'clientDataJSON' => 'eyJ0eXBlIjoid2ViYXV0aG4uY3JlYXRlIiwiY2hhbGxlbmdlIjoidXdGS2ZyRTk3Qm1yWWFmUjhUZW5kUjJKbWxkekVlQ3paRTFnL0FhYm03bz0iLCJvcmlnaW4iOiJodHRwOi8vbG9jYWxob3N0IiwiY2lkX3B1YmtleSI6bnVsbH0=',
+                'attestationObject' => 'pGNmbXRoZmlkby11MmZnYXR0U3RtdKJjc2lnWEgwRgIhAL2a2xVBg_4Ooc-m27dxItzeUsROR7PLh2wHa0ZTerhYAiEA9trBQ8Yr6MPPdeNaN4BE8fuR4aV2iL8UL95JfB4F-khjeDVjgVkBJzCCASMwgcmgAwIBAgIhARDyEPt8s80lRZ3lTdjXIo0Dp3dfBJd1nOqwKNDeOPNOMAoGCCqGSM49BAMCMAAwIBcNMjIwMTAxMDEwMTAxWhgPMjEyMjAxMDEwMTAxMDFaMAAwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQG1hpCGK0y-a68p55Rma6H3pOZdNEbEH_pt-yVaZNUXorWH479PxHoyOQhYKxf4-vLxEmG47FsPCi-dgkG7RJnoxIwEDAOBgNVHQ8BAf8EBAMCAqQwCgYIKoZIzj0EAwIDSQAwRgIhAIXIqNEsaurdLaUiLG5_srVUw8fZZyJ268Hh8iFp3Xb2AiEA-v_2ik8SC8_EhQzN4RkgHRseGr-y0DymcIbdrpODjYpoQXV0aERhdGGlZHJwaWT2ZWZsYWdzAGhhdHRfZGF0YaNmYWFndWlk9mpwdWJsaWNfa2V59m1jcmVkZW50aWFsX2lk9mhleHRfZGF0YfZqc2lnbl9jb3VudABoYXV0aERhdGFYjoYFuLoIwg1C-eRVFRhxiW0ODemAWWKG-3NtEe7AE-KkQQAAAAAAAAAAAAAAAAAAAAAAAAAAAAx2aXJ0dWFsa2V5MTGkAQIDJiFYIAbWGkIYrTL5rrynnlGZrofek5l00RsQf-m37JVpk1ReIlggitYfjv0_EejI5CFgrF_j68vESYbjsWw8KL52CQbtEmc',
+            ],
+            'user' => [
+                'displayName' => $user->display_name,
+                'id' => $user->id,
+                'name' => $user->username,
+            ],
+            'transports' => ['usb'],
+        ];
+
+        $reqJson = json_encode($reqValue);
```

Comment:
You'll want to remove this, too, as it should be the array being passed, not a JSON-encoded string of the array.
```suggestion
```